src/pages/messages/components/NviStatusPage.tsx

```diff
@@ -0,0 +1,126 @@
+import { Organization } from '../../../types/organization.types';
+import { isValidUrl } from '../../../utils/general-helpers';
+import { getLanguageString } from '../../../utils/translation-helpers';
+
+interface NviStatusPageProps {
+  activeYear: number | string;
+}
+
+export const NviStatusPage = ({ activeYear }: NviStatusPageProps) => {
+  const { t } = useTranslation();
+  const user = useSelector((store: RootState) => store.user);
+
+  const organizationQuery = useFetchOrganization(user?.topOrgCristinId);
+  const institution = organizationQuery.data;
+
+  const nviQuery = useFetchNviCandidates({ size: 1, aggregation: 'all', year: activeYear });
+  const aggregationKeys = Object.keys(nviQuery.data?.aggregations?.organizationApprovalStatuses ?? {});
+  const aggregationKey = aggregationKeys.find((key) => isValidUrl(key));
+  const nviAggregations = nviQuery.data?.aggregations?.organizationApprovalStatuses[aggregationKey ?? ''] as
+    | OrganizationApprovalStatusDetail
+    | undefined;
+
+  return (
+    <BackgroundDiv>
+      <Typography variant="h1" gutterBottom>
+        {t('search.reports.institution_nvi')}
+      </Typography>
+
+      <TableContainer>
+        <Table size="small">
+          <TableHead>
+            <TableRow sx={{ whiteSpace: 'nowrap' }}>
+              <TableCell>{t('registration.contributors.department')}</TableCell>
+              <TableCell>{t('tasks.nvi.status.New')}</TableCell>
+              <TableCell>{t('tasks.nvi.status.Pending')}</TableCell>
+              <TableCell>{t('tasks.nvi.status.Approved')}</TableCell>
+              <TableCell>{t('tasks.nvi.status.Rejected')}</TableCell>
+              <TableCell>{t('common.total_number')}</TableCell>
+              <TableCell>{t('tasks.nvi.publication_points')}</TableCell>
+              <TableCell>{t('tasks.nvi.status.Dispute')}</TableCell>
+              <TableCell>
+                <Box component="span" sx={visuallyHidden}>
+                  {t('tasks.nvi.show_subunits')}
+                </Box>
+              </TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {institution && <OrganizationTableRow organization={institution} aggregations={nviAggregations} />}
+          </TableBody>
+        </Table>
+      </TableContainer>
+    </BackgroundDiv>
+  );
+};
+interface OrganizationTableRowProps {
+  organization: Organization;
+  aggregations: OrganizationApprovalStatusDetail | undefined;
+  level?: number;
+}
+
+const OrganizationTableRow = ({ organization, aggregations, level = 0 }: OrganizationTableRowProps) => {
+  const { t } = useTranslation();
+  const [expanded, setExpanded] = useState(level === 0);
+
+  const hasSubUnits = organization.hasPart && organization.hasPart.length > 0;
+  const thisAggregations = aggregations?.organizations?.[organization.id];
+
+  return (
+    <>
+      <TableRow sx={{ bgcolor: level % 2 === 0 ? '#f6f1dc' : '#faf7eb' }}>
```

Comment:
Må avklare med designer hva vi skal med fargene her. er ikke så hypp på å legge inn hardkodede verdier bare for denne tabellen